index.html

```diff
@@ -0,0 +1,237 @@
+          <hr class="menu-line" />
+        </li>
+        <li>
+          <a href="#contact-form" class="nav-link">Contact</a>
+          <hr class="menu-line" />
+        </li>
+      </ul>
+    </section>
+
+    <!-- page 2 -->
+    <div class="page2 about-page">
+      <!-- about -->
+      <section class="about">
+        <h3 class="sub-title">"Hello! sharing the world!"</h3>
+        <h1 class="about-title">
+          WORLD CONFerance <br />
+          ALERTS 2022
+        </h1>
+        <p class="paragraph1 color">
+          Full-stack Web Development courses offered through Coursera equip learners with knowledge in building web, hybrid, and mobile solutions; understanding front-end frameworks such as Bootstrap 4 and Angular; implementing NoSQL databases using MongoDB; creating the best user
+           experience regardless of device; the basics of ...
+        </p>
+        <h2 class="contact">
+          Please contact us per Email for any further questions<br />
+          about CC Global Summi 2015!<br />
+          <a href="#">ccsummit@cckorea.org</a>
+        </h2>
+      </section>
+
+      <!-- global summit -->
+      <section class="global-summit">
+        <h2 class="">
+          Creative Commons<br />
+          Global Summit 2015 Logo
+        </h2>
+        <hr class="line1" />
+        <p class="address">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit onsectetur
+          adipisicing elit
+        </p>
+        <img
+          class="about-img"
+          src="./img/about_2013.png"
+          alt="2013 comferance image"
+        />
+        <img
+          class="about-img"
+          src="./img/about_2011.png"
+          alt="2011 comferance image"
+        />
+      </section>
+
+      <!-- partners -->
+      <section id="partner" class="partner">
+        <h2 class="program-title">Partner</h2>
+        <hr class="line1" />
+        <ul class="partners">
+          <li>
+            <img
+              class="company-icon"
+              src="./img/mozilla.png"
+              alt="mozila icon"
+            />
+          </li>
+          <li>
+            <img
+              class="company-icon"
+              src="./img/googlekorea.png"
+              alt="mozila icon"
+            />
+          </li>
+          <li>
+            <img class="company-icon" src="./img/naver.png" alt="mozila icon" />
+          </li>
+          <li>
+            <img class="company-icon" src="./img/kakao.png" alt="mozila icon" />
+          </li>
+          <li>
+            <img
+              class="company-icon"
+              src="./img/airbnb.png"
+              alt="mozila icon"
+            />
+          </li>
+        </ul>
+      </section>
+
+      <!-- footer -->
+      <footer class="footer">
+        <img class="logo-img" src="./img/logo.png" alt="logo image" />
+        <p class="contact">
+          2015 Creative Commons Korea.<br />
+          Some Rigths Reserved.
+        </p>
+      </footer>
```

Comment:
- [ ] Your footer looks good and similar to the template on the mobile version of the 'About' page, but on the desktop version of the 'Home' page the footer is missing and the desktop of the 'About' page is slightly different with the partner section and white background color when on the template it has a dark background color. Kindly make the changes so the footer looks similar to the template on those pages.
 - Your About desktop:
 - 
<img width="944" alt="image" src="https://user-images.githubusercontent.com/61919617/210398428-6ceabfc4-70ff-44b5-bd59-fa5808a25410.png">

 - Template About desktop:
 - 
<img width="882" alt="image" src="https://user-images.githubusercontent.com/61919617/210398520-259b7c77-03f9-4da0-9b2d-9fc55cc9c2d4.png">

 - Your Home desktop:
 - 
<img width="1145" alt="image" src="https://user-images.githubusercontent.com/61919617/210398706-8506e53d-df0e-4575-b785-3c8c145f4e4d.png">

 - Template Home desktop:
 - 
<img width="1213" alt="image" src="https://user-images.githubusercontent.com/61919617/210398824-0d69ab7f-ac3a-4b33-8c2d-9c157a8ce871.png">

Reply:
done